man/systemd-cryptenroll.xml

```diff
@@ -299,6 +299,17 @@
         signatures likely will validate against pre-existing certificates.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-with-pin=</option><replaceable>BOOL</replaceable></term>
+
+        <listitem><para>When enrolling a TPM2 device, controls whether to require the user to enter a PIN
+        when unlocking the volume in addition to PCR binding, based on TPM2 policy authentication. The pin
+        needs to be 4-32 characters long. Defaults to <literal>no</literal>. Note that incorrect PIN entry
```

Comment:
maybe say explicitly that "despite the concept being named PIN this may include any character, not just numbers".